hermes/bin/src/runtime/host/hermes/hash/mod.rs

```diff
@@ -30,8 +34,13 @@ impl Host for HermesState {
     fn blake2b(
         &mut self, buf: Bstr, outlen: Option<u8>, key: Option<Bstr>,
     ) -> wasmtime::Result<Result<Bstr, Errno>> {
-        // WIP
-        todo!()
+        // Recheck this implementation might need to use Blake2sMac256 for adding the key
+        let mut v = Blake2bVarCore::new_with_params(salt, persona, key_size, output_size);
```

Comment:
I would rather move all this code under some free function `blake2b(...)` and call it inside `HermesState::blake2b`.
So the unit test would be much simpler and will cost for us nothing, you will not need to setup a `Hermes` instance,  wasm module etc.

And to proper test how it is working under the WASM context, we will would rather write integration tests, but it is another story.